tools/omfwd.c

```diff
@@ -96,6 +97,10 @@ typedef struct _instanceData {
 	int iKeepAliveProbes;
 	int iKeepAliveTime;
 	uchar *gnutlsPriorityString;
+	/*  0 means:  IP_FREEBIND is disabled
+	1 means:  IP_FREEBIND enabled + warning disabled
```

Comment:
I would prefer #define's here for readability. It also looks like the comment is in error ;-)

Reply:
Done, the #defines are already in runtime/net.h and well-commented, so I have removed this comment altogether from the copy & paste I did from imudp.c cf original commit 5f12c9b5 there.